server/src/coreV2/user/dtos/create-user.input.ts

```diff
@@ -0,0 +1,38 @@
+import { Field, InputType } from '@nestjs/graphql';
+
+import { IsNotEmpty, IsOptional, IsString } from 'class-validator';
+
+@InputType()
+export class CreateUserInput {
+  @IsString()
+  @IsNotEmpty()
+  @Field()
+  @IsOptional()
+  firstName: string;
```

Comment:
@gitstart-twenty When `IsOptional` is provided the proper typescript type should be used:
```typescript
firstName?: string;
```